helios-client/src/main/java/com/spotify/helios/common/descriptors/RolloutOptions.java

```diff
@@ -101,10 +101,9 @@ public Builder toBuilder() {
         .setIgnoreFailures(ignoreFailures);
   }
 
-
   /**
    * Return a new RolloutOptions instance by merging this instance with another one.
-   * The other instance is assumed to have no null-valued fields.
+   * Throws NPE if the attribute in both instances is null.
```

Comment:
nitpick: use `@throws`, and also I think you mean to say "any attribute in either instance" not "the attribute"

Reply:
done